packages/ui-inputs/src/button.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
-import {useTheme, createUseStyles, theming} from './theme';
-import {Button as AntButton} from 'antd';
-import { ReactNode } from 'react';
+import { createUseStyles, theming, useTheme } from './theme';
```

Comment:
General issue
1. please check this in ie11, safari, chrome, firefox.
2. to me it doesn't look the same, but I might be wrong
3. in chrome (maybe in others as well) - it seems that the text and the icon doesn't  align correctly (the icon is higher?)

Maybe I'm wrong, can you double check this one?

![image](https://user-images.githubusercontent.com/4751797/70083911-fb08be80-1615-11ea-834c-c7ed28df31d1.png)

Reply:
I guess it might happen because the font is rendered a bit differently, I've added line-height to make label constant 24px height, but your screenshots look fine to me (I even measured them with a screen ruler 🙂).
Anyway, does it looks better now?
![image](https://user-images.githubusercontent.com/4395889/70122018-7ef39280-1678-11ea-857e-d755984ca91e.png)

Reply:
👍